### ISSUE TITLE
posts can only be saved as draft

### ISSUE THREAD

**Issue by Geczy** [shadcn/taxonomy#10]
publish coming soon?

**Comment by shadcn** [comment]
Yep. It's in progress. 

Note: this is a demo app. The content is all temp. Do not use it to build your blog 😬

**Comment by Geczy** [comment]
Yeah it's missing a lot of features still :D i wanted to get something live soon
